docs/api/go/getting-started.md

```diff
@@ -113,4 +113,4 @@ Also, the following examples demonstrate how to:
 9. [Use paymaster to pay fee with token](examples/custom-paymaster/use-paymaster.md).
 
 Full code for all examples is available [here](https://github.com/zksync-sdk/zksync2-examples/tree/main/go). Examples are configured to
-interact with `zkSync Era` and `Goerli` test networks.
+interact with `zkSync Era`, `Sepolia` and `Goerli` test networks.
```

Comment:
Did we make configuration changes there for Sepolia?

Reply:
https://github.com/zksync-sdk/zksync2-examples/commit/92de749b21e4a5277bc7e8c88774cd3de49e8fbb

Reply:
It looks like we replaced goerli with sepolia so why do we say `zkSync Era`, `Sepolia` and `Goerli`? Should it be just Sepolia and no Goerli?